helpers/createAgTokenSideChainMultiBridge.sh

```diff
@@ -0,0 +1,128 @@
+#! /bin/bash
+
+source lib/utils/helpers/common.sh
+
+function usage {
+  echo "bash createAgTokenSideChainMultiBridge.sh <chain> <chainUri> <stableName>"
+  echo ""
+  echo -e "chainId: chain to run the script on"
+  echo -e "chainUri: rpc uri for the chain"
+  echo -e "stableName: name of the stable token"
+  echo ""
+}
+
+function main {
+    if [[ $# -ne 4 ]]; then
+        usage
+        exit 1
+    fi
+    chain=$1
+    chainUri=$2
+    stableName=$3
+
+    if [ -z "$chain" ] || [ -z "$chainUri" ] || [ -z "$stableName" ]; then
+        echo "Missing arguments"
+        exit 1
+    fi
+
+    if [ ! -f .env ]; then
+        echo ".env not found!"
+        exit 1
+    fi
+    source .env
+
+    export CHAIN_ID=$chain
+    export STABLE_NAME=$stableName
+
+    echo ""
+    echo "Running deployment on chain $chain for stable token $stableName"
+
+    FOUNDRY_PROFILE=dev cd lib/angle-tokens && forge script DeployAgTokenSideChainMultiBridge --rpc-url $chainUri --verify --broadcast && cd ../..
+
+    if [ $? -ne 0 ]; then
+        echo ""
+        echo "Deployment failed"
+        exit 1
+    fi
+
+    echo ""
+    echo "Deployment successful"
+
+    echo ""
```

Comment:
Let's add test files to check that everything is well setup and do bridge transaction by faking LZ